service/controller/v18/cloudconfig/worker_template.go

```diff
@@ -56,7 +61,7 @@ func (c *CloudConfig) NewWorkerTemplate(ctx context.Context, customObject v1alph
 		}
 	}
 
-	return newCloudConfig.Base64(), nil
+	return newCloudConfig.String(), nil
```

Comment:
Why does the interface change now? What implications does this have?

Reply:
It is changed, because in small config we are using ignition with `append`, so operator ignition is merged with cloudconfig ignition. And it doesn't support base64 itself